functions/src/modules/Files.functions.ts

```diff
@@ -31,36 +33,34 @@ export const importFilesFromStorage = functions.storage
     const list = helpers.extractListFromFilename(fileName);
 
     const fileRef = db.ref(`/files/${list}/${fileName}`);
+    const file = (await fileRef.once('value')).val();
 
     fileRef.set({
-      mp3Uploaded: isValidFile[3] ? true : false,
-      flacUploaded: isValidFile[5] ? true : false,
+      mp3Uploaded: isValidFile[3] ? true : file.mp3Uploaded,
+      flacUploaded: isValidFile[5] ? true : file.flacUploaded
     });
+
     return 1;
   });
 
-export const HandleDeletedFiles = functions.storage
-  .bucket(`original${functions.config().storage['root-domain']}`)
+export const handleOriginalFileDeletion = functions.storage
+  .bucket(originalBucket)
   .object()
   .onDelete(async object => {
     const filePath = object.name;
     const nameParts = checkValidFile(filePath);
-    if (!nameParts) return -1;
+    if (!nameParts) {
+      console.warn(`File: "${filePath}" was deleted.`);
```

Comment:
Warning is not about the fact that the file was deleted, but about the fact that we ignore it for some reason.

Reply:
This function is triggered when files are deleted, there's no way to find out the reason that caused the file to be deleted.